docker/orthanc/Dockerfile

```diff
@@ -37,17 +40,48 @@ COPY ./pixl_dcmd/ /pixl_dcmd
 RUN --mount=type=cache,target=/root/.cache \
     /.venv/bin/pip install --no-cache-dir --force-reinstall --no-deps ./pixl_dcmd
 
-COPY ./orthanc/orthanc-raw/plugin/pixl.py /etc/orthanc/pixl.py
+ARG PIXL_USER_GID
+RUN groupadd --gid $PIXL_USER_GID pixl && usermod -a -G pixl orthanc
+USER orthanc
+
+# Because the volume contents inherit the ownership of the mountpoint if it already exists,
+# creating and chowning at this point ensures that this happens.
+# See https://devops.stackexchange.com/a/4542 although the docs no longer seem to mention this.
+# The VOLUME instruction is not needed because this is specified by the docker compose file.
+ARG ORTHANC_DATA_VOLUME=/var/lib/orthanc/db/
+RUN mkdir "$ORTHANC_DATA_VOLUME" && chown orthanc:orthanc "$ORTHANC_DATA_VOLUME"
+
+
+# Orthanc-raw
+# =================================================================================================
+FROM pixl_orthanc_base AS pixl_orthanc_raw
+
+COPY --chown=orthanc:orthanc ./orthanc/orthanc-raw/plugin/pixl.py /etc/orthanc/pixl.py
 
-# Orthanc can't substitute environment veriables as integers so copy and replace before running
 ARG ORTHANC_RAW_MAXIMUM_STORAGE_SIZE
 ARG ORTHANC_RAW_JOB_HISTORY_SIZE
 ARG ORTHANC_CONCURRENT_JOBS
 ARG PIXL_DICOM_TRANSFER_TIMEOUT
-COPY ./orthanc/orthanc-raw/config /run/secrets
-RUN sed -i "s/\${ORTHANC_RAW_MAXIMUM_STORAGE_SIZE}/${ORTHANC_RAW_MAXIMUM_STORAGE_SIZE:-0}/g" /run/secrets/orthanc.json
-RUN sed -i "s/\${ORTHANC_RAW_JOB_HISTORY_SIZE}/${ORTHANC_RAW_JOB_HISTORY_SIZE:-100}/g" /run/secrets/orthanc.json
+
+# Orthanc can't substitute environment veriables as integers so copy and replace before running
+COPY --chown=orthanc:orthanc ./orthanc/orthanc-raw/config /run/secrets
+RUN <<EOF
+sed -i "s/\${ORTHANC_RAW_MAXIMUM_STORAGE_SIZE}/${ORTHANC_RAW_MAXIMUM_STORAGE_SIZE:-0}/g" /run/secrets/orthanc.json
+sed -i "s/\${ORTHANC_RAW_JOB_HISTORY_SIZE}/${ORTHANC_RAW_JOB_HISTORY_SIZE:-100}/g" /run/secrets/orthanc.json
+sed -i "s/\${ORTHANC_CONCURRENT_JOBS}/${ORTHANC_CONCURRENT_JOBS:-5}/g" /run/secrets/orthanc.json
+sed -i "s/\${ORTHANC_RAW_STABLE_SECONDS}/${PIXL_DICOM_TRANSFER_TIMEOUT:-600}/g" /run/secrets/orthanc.json
+EOF
+
+ENV PYTHONPATH=/.venv/lib64/python3.11/site-packages/
+
+
+# Orthanc-anon
+# =================================================================================================
+FROM pixl_orthanc_base AS pixl_orthanc_anon
+
+COPY --chown=orthanc:orthanc ./orthanc/orthanc-anon/plugin/pixl.py /etc/orthanc/pixl.py
+COPY --chown=orthanc:orthanc ./orthanc/orthanc-anon/config /run/secrets
+
 RUN sed -i "s/\${ORTHANC_CONCURRENT_JOBS}/${ORTHANC_CONCURRENT_JOBS:-5}/g" /run/secrets/orthanc.json
```

Comment:
`ORTHANC_CONCURRENT_JOBS` doesn't seem to be declared as a build ARG for `pixl_orthanc_anon`. Is it just reverting to the given default value?